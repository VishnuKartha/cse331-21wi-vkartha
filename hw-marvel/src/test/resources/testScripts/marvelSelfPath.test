# Tests that self paths from characters to itself have the desired output

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/marvel.tsv

LoadGraph marvel marvel.tsv

FindPath marvel DRAGONMEGAS,_JESSIE DRAGONMEGAS,_JESSIE