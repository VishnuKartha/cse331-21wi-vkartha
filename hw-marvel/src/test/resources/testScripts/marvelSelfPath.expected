# Tests that self paths from characters to itself have the desired output

# To run this test, the marvel dataset must exist at
# hw-marvel/src/main/resources/data/marvel.tsv

loaded graph marvel

path from DRAGONMEGAS, JESSIE to DRAGONMEGAS, JESSIE: